### DISCUSSION TITLE
Can i add 2 hours to {{ getDate (getDate (slot item @root.slots.PUETA) "YYYY-MM-DDTHH:mm:ss.0000000\Z") "MMMM DD, YYYY h:mm a" 3 }}

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3931] [Q&A]
I have this formula inside PnP Modern Search web part:-

`{{ getDate (getDate (slot item @root.slots.PUETA) "YYYY-MM-DDTHH:mm:ss.0000000\Z") "MMMM DD, YYYY h:mm a" 3 }}
`

to format a date time, and allow it to follow the time zone of the site.. my question is how i can add 2 hours to the generated date/time from the above formula?

Thanks

**Comment by wobba** [comment]
As per https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#getdate the function support what moment.js support.

**Reply by mvcsharepointdev** [reply]
@wobba but I cannot see how i can add 2 hours ? is this possible?
thanks

**Reply by wobba** [reply]
@mvcsharepointdev I don't know. Look at moment.js documentation.
